### PR TITLE
DeltaStation Themifying!

### PR DESCRIPTION
**Pull request by Vect0r2**
## About The Pull Request

I decided for my first mapping challenge to give the DeltaStation heads some edited rooms. They now have themed for whatever head they are, and the HOPs room has been built up from scratch with a larger bedroom. 
ALSO PHOTOS:
caps office
CE
CMO
hop
HOS
RD

### Mapping March

Ckey to recieve rewards: Vect0r

## Why It's Good For The Game

It makes it look nicer (I think), and gives Delta unique flavor compared to Meta, and any other station that has basic head rooms. Also, the hop should have a bedroom that is bigger than a 2x3 room.

## Changelog

:cl:
add: Expanded Delta Station HOPs office and added themed carpets to all the important heads! Have fun!
/:cl:
(not sure if that's the right tag ngl)

### CODE REVIEW

**Comment by Helg2** [comment]
It looks like shit.

**Comment by Dorsisdwarf** [comment]
Do you have an in-game version of the screenshots? Map editor screenshots always look off.

**Comment by SomeoneYouProbablyKnow** [comment]
They are all a bit too busy going off the map screenshots (Especially the glowing hos floors). 

And you sacrifice some QoL for that busyness (like with that long hallway to HoP; Although we could argue this as an interesting map change, since it's a big chokepoint) 

Tl;Dr Sometimes less is more. Keep at it

**Comment by Vect0r2** [comment]
updated with images in game.

**Comment by LemonInTheDark** [comment]
Don't want to steal gbp from such a hardworking up-and-comer

**Comment by Vect0r2** [comment]
Hell yeah

**Comment by ArthuroPep** [comment]
Give red Carpet to the HoS! Theirs looks like an evil AI chamber

**Comment by Vect0r2** [comment]
Yeah I should.

**Comment by san7890** [comment]
image

very important that you fix this

**Comment by Vect0r2** [comment]
> image
> 
> very important that you fix this, the coordinates 141,118 should correspond to the x,y coordinates of your map file
> 
> ```
> /datum/unit_test/mapload_space_verification
>   Error: Space turf found in non-allowed area (/area/station/command/heads_quarters/hop) at Head of Personnel's Office (141,118,2)! Please ensure that all space turfs are in an /area/space!
>   	FAILURE #1: Space turf found in non-allowed area (/area/station/command/heads_quarters/hop) at Head of Personnel's Office (141,118,2)! Please ensure that all space turfs are in an /area/space! at code/modules/unit_tests/mapload_space_verification.dm:34
> Error: FAIL /datum/unit_test/mapload_space_verification 0.6s
> ```
> 
> Mark as ready for review when this has been addressed.

Ok! Got IRL stuff to do, but I will fix it when I got time.

**Comment by Vect0r2** [comment]
Nevermind. I don't got a life. fixed!

**Review by san7890 — CHANGES_REQUESTED** [review]
you've got even more CI issues. if you can't figure it out (the coordinates should line up exactly to the ones on your map file), shoot a link (i'll help you, it's this: https://github.com/tgstation/tgstation/actions/runs/4432366209/jobs/7776512481?pr=73891) to people in either coding/mapping general to the failing CI run and then mark this PR ready for review when those have been addressed.

**Comment by Wallemations** [comment]
What happened to the rug under lamaar's tank? It looks like it's both plating and a rug at the same time

**Comment by Vect0r2** [comment]
Fuck, ok, gonna get this ready-ish today

**Comment by Pumpkinoe** [comment]
image
the corridor of hell

**Comment by Pumpkinoe** [comment]
image
the floor fax machine, to encourage HoPs to lie on the ground to interact with their machinery

**Comment by Pumpkinoe** [comment]
you forgot to put an ID/PDA painter in the HoPs office (Extremely important for his job - as in he literally cannot do it without one)

**Comment by Pumpkinoe** [comment]
it also looks like you've taken away the HoP's paper bin
No More Paperwork...

**Comment by Pumpkinoe** [comment]
image
i must question the random shuttle door as well

**Comment by kinneb** [comment]
youve got doubles: 
image

**Comment by EOBGames** [comment]
I just have to be honest, I don't think any of this is really improving on what's already there. You've got an unfortunate mix of too busy (HoS and RD in particular just have too many varying tile types) and too empty (the HoP office really stands out here- you've got a big empty square for an office).

I'd encourage you to stick at mapping, but I just don't think this particular PR is doing much to make delta better.
